day2/자릿수-더하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(n) {
+    let str = String(n);
+    let int = str.split('');
+    let nmap = int.map(Number);
+    let answer = 0;
+        for(let i = 0; i < nmap.length; i++) {
+        answer += nmap[i];
```

Comment:
```suggestion
        for(let i = 0; i < nmap.length; i++) {
            answer += nmap[i];
```
가독성을 위해 들여쓰기 꼭 지켜주세요